src/api-reference/fi-service/financial-integration-service.markdown

```diff
@@ -0,0 +1,1124 @@
+                                              "paymentRelevantLineItems":[],
+                                              "postingDate":"2018-03-02"
+                                          },
+                                          {
+                                              "companyId":"0800",
+                                              "documentNumber":"0123478",
+                                              "fiscalYear":"2018",
+                                              "paymentRelevantLineItems":[],
+                                              "postingDate":"2018-03-02"
+                                          }
+                                      ],
+                                  "systemMessages":[]
+                              }
+                          ]
+		}
+```
+
+_Example Response with Sucsess and Failure_
+
+```
+[
+          {
+              "code": 0,
+              "docId": "0c06ab044834454d91f83cbd7b6431d2",
+              "systemId": null,
+              "postingConfirmationResult": "SUCCESS",
+              "errorMessage": "",
+              "detailMessage": ""
+          },
+          {
+              "code": 116,
+              "docId": "3331dbeb8e2240ffad7ab5b69492722a",
+              "systemId": "",
+              "postingConfirmationResult": "FAILURE",
+              "errorMessage": "This document does not exist.",
+              "detailMessage": ""
+          }
+  ]
+```
+
+## Response Codes
+
+### Success Codes
+
+| Code | Message | Information |
+| --- | --- | --- |
+| 200 | OK | Your request has been processed, please refer to the Financial Integration code table below for more details. |
+
+### Financial Integration Codes
+
+The Financial Integration Service will return particular codes based on the success and failure of individual records for Acknowledging and Posting Confirmation of documents.
+
+| Code | Description | Category |
+| --- | --- | --- |
+| 0 | Successfully processed | Any |
+| 99 | System Id in request does not match system id in FI database | Any |
+| 101 | This document was previously acknowledged | Acknowledge |
+| 102 | This document has been recalled | Acknowledge |
+| 103 | This document is not ready | Acknowledge |
+| 104 | This document does not exist in FI database | Acknowledge |
+| 105 | This document is not of type (expense, invoice, cashadvance, payroll) | Acknowledge |
+| 111 | This document has not been acknowledged | Posting |
+| 112 | This document has been recalled | Posting |
+| 113 | Confirmation has been posted for this document | Posting |
+| 114 | Document is not in a known state | Posting |
+| 115 | This document is not of type (expense, invoice, cashadvance, payroll) | Posting |
+| 116 | This document does not exist in FI | Posting |
+| 198 | Invalid request - this same request will not work if tried again. | Posting |
+| 199 | Unknown error - please try again later | Any |
+
+### Failure Codes
```

Comment:
We should make a clear distinction between HTTP Status Codes and API codes. Documentation template is your friend.

Reply:
Take a look at the revised version Imene submitted today addressing your input (last commit 991304a). For the entityID and CompanyOutask ID, Teresa’s team are working on a v2 of this API to hide those details to the customers.